### ISSUE TITLE
[Feature] Add Phosphor Icons

### ISSUE THREAD

**Issue by gps-lasrol** [AathifMahir/MauiIcons#123]
**Is your feature request related to a problem? Please describe.**
My application requires us to use Phosphor Icons in our project.

**Describe the solution you'd like**
I have created a package similar to the existing packages with the Phosphor Icons implementation. 
If you want, i can provide a PR for this. 

**Describe alternatives you've considered**
If you do not want this, i will only build this package for my own use. 

**Additional context**
[Phosphor Icons](https://phosphoricons.com)
[Implementation](https://github.com/gps-lasrol/MauiIcons/tree/phospho-icons)

**Comment by AathifMahir** [comment]
> **Is your feature request related to a problem? Please describe.**
> My application requires us to use Phosphor Icons in our project.
> 
> **Describe the solution you'd like**
> I have created a package similar to the existing packages with the Phosphor Icons implementation. 
> If you want, i can provide a PR for this. 
> 
> **Describe alternatives you've considered**
> If you do not want this, i will only build this package for my own use. 
> 
> **Additional context**
> [Phosphor Icons](https://phosphoricons.com)
> [Implementation](https://github.com/gps-lasrol/MauiIcons/tree/phospho-icons)
> 

Of course, we can merge it to Main Library, Please create a Pull Request from your fork, I'll look through changes and improvements that needed to be done for module and push a official Phosphor Icon.

**Comment by gps-lasrol** [comment]
I have opened a PR :)
